pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -394,42 +397,54 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
 
+	nsReg := linearNS(nodesCount)
+	nsReg, err := s.domain.Nodes[0].NSRegistryClient.Register(ctx, nsReg)
+	require.NoError(t, err)
+
 	var nsesReg [nodesCount]*registry.NetworkServiceEndpoint
 	for i := 0; i < nodesCount; i++ {
+		nsesReg[i] = &registry.NetworkServiceEndpoint{
+			Name:                fmt.Sprintf("endpoint-%v", i),
+			NetworkServiceNames: []string{nsReg.GetName()},
```

Comment:
Probably just `.Name` is more common for this in tests.

Reply:
done